### PR TITLE
feat: rnf invoice data format 3, grouped tax

### PR DESCRIPTION
**Pull request by AdamDowson**
## Description of the changes

rnf invoice data type 3.0, grouping of tax

## Link to Jira

https://app.asana.com/0/1194620076282706/1196258233315537

### CODE REVIEW

File: packages/data-format/src/format/rnf_invoice/rnf_invoice-0.0.3.json [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+          },
+          "unitPrice": {
+            "type": "string",
+            "pattern": "^\\d+$"
+          },
+          "discount": {
+            "type": "string",
+            "pattern": "^\\d+$"
+          },
+          "tax": {
```
Comment (AdamDowson) [inline packages/data-format/src/format/rnf_invoice/rnf_invoice-0.0.3.json:133]: This is the only changes vs 0.0.2

**Review by yomarion — APPROVED** [review]
Good now !

File: packages/data-format/test/data/example-invalid-0.0.3.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      "miscellaneous": {
+        "note": "payment before chrismas"
+      }
+    },
+  
+    "miscellaneous": {
+      "manufacturerCompany": "Victor Hugo & Co.",
+      "deliveryCompany": "Gavroche Express"
+    }
+}
```
Comment (vnnkl) [inline packages/data-format/test/data/example-invalid-0.0.3.json:86]: no newline

**Review by vnnkl — APPROVED** [review]

**Review by rittme — APPROVED** [review]

**Review by benjlevesque — APPROVED** [review]

File: packages/data-format/src/format/rnf_invoice/rnf_invoice-0.0.3.json [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+            "description": "Tax information about the invoice",
+            "type": "object",
+            "additionalProperties": false,
+            "required": ["amount", "type"],
+            "properties": {
+              "amount": {
+                "type": "number"
+              },
+              "type": {
+                "type": "string"
```
[the remark this answers is not in the document]
Comment (AdamDowson) [reply packages/data-format/src/format/rnf_invoice/rnf_invoice-0.0.3.json]: Yes, good point. I am using the naming convention:

`percentage` and `fixed`

**Pull request merged**